privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -216,11 +329,17 @@ def train_meta_classifier(
             metrics=["accuracy"],
         )
 
+        # keras functional API provides a verbose variable ranging from {0, 1, 2}.
+        # logging uses levels in our case corresponding to numeric values from {30, 20, 10}.
+        # We can therefore convert our self.logger.level to the appropriate verbose value in the following manner:
```

Comment:
This would not necessarily be a comment that you keep for a piece of software. It's rather information for yourself, so you might leave that out.

Reply:
We wrote this comment, because the line which follows needs explanation as it would appear a bit random otherwise.